CreeDictionary/DatabaseManager/xml_entry_lemma_finder.py

```diff
@@ -29,7 +29,9 @@ def extract_fst_lemmas(
     logger = DatabaseManagerLogger(__name__, verbose)
     logger.info("Determining lemma analysis for (xml_lemma, xml_pos, xml_lc) tuples...")
 
-    xml_lemma_pos_lc_to_analysis = dict()  # type: Dict[Tuple[str, str, str], ConcatAnalysis]
+    xml_lemma_pos_lc_to_analysis = (
+        dict()
+    )  # type: Dict[Tuple[str, str, str], ConcatAnalysis]
```

Comment:
My IDE did some formatting automatically with black.

Reply:
Haha, black HATES same line comments. Which brings up a good point: why _isn't_ this a regular type annotation?